### DISCUSSION TITLE
Support for text bundles/textpacks

### DISCUSSION THREAD

**Discussion by swduncan** [getpelican/pelican#3418] [Q&A]
Is there any support or plugin for textbundles or textpacks? I see this discussion from 10 years ago: https://github.com/getpelican/pelican/issues/1482, but if the plugin is in Github anywhere I have not been able to find it.

**Comment by justinmayer** [comment]
According to said plugin's author, it seems the purpose of the plugin has been obviated by Pelican 3.5+'s ability to `{attach}` files, so he removed the plugin: https://github.com/getpelican/pelican-plugins/pull/485

Barring that approach, I think the next option would be to write a custom plugin to process the textbundles/textpacks, perhaps as a new `Reader`. (See: https://docs.getpelican.com/en/latest/internals.html)

**Reply by swduncan** [reply]
I see that the Attach method served the author's purpose, but it doesn't seem to allow Pelican to use textbundles as content. Apparently the author wasn't using them to take content in but to organize it once it was in. 

I'm looking for a way to get the textbundles output by Ulysses, and process those as posts.

**Reply by justinmayer** [reply]
I updated my answer with the next-best option, which is write a custom plugin that implements the behavior you're describing.
